app/Models/Post.php

```diff
@@ -371,6 +384,10 @@ public function toBlinkPayload()
             'source_details' => $this->source_details,
             'details' => $this->details,
             'referrer_user_id' => $this->referrer_user_id,
+            'group_id' => $this->group_id,
+            'group_name' => isset($group) ? $group->name : null,
+            'group_type_id' => isset($group) ? $group->group_type_id : null,
+            'group_type_name' => isset($group) ? $group->group_type->name : null,
```

Comment:
Would 'grouping' (ha) this and the below list of attributes in `getReferralPostEventPayload` into like a `getGroupPayload` -or something- helper be useful or overkill? (Maybe useful for the Signup as well?)

Reply:
I like this idea.. maybe this could be a static method on the Group model, since we won't always have a group on a signup or a post -- but would still want to return null values for all of these array keys.

Reply:
Ooh, that makes good sense to me!

Reply:
I'm going to hold off on this for now.. the one thing I wouldn't be thrilled about is the `group_` fields appearing after the `created_at` and `updated_at` fields when viewing the events with Customer.io. Minor, but can always revisit.

Reply:
Ah, testing this and Customer.io orders the payload properties alphabetically anyway. I'll add this in a cleanup PR.